driver-core/src/test/functional/com/mongodb/connection/netty/NettyStreamSpecification.groovy

```diff
@@ -68,24 +67,6 @@ class NettyStreamSpecification extends Specification {
         thrown(MongoSocketOpenException)
     }
 
-    @IgnoreIf({ getSslSettings().isEnabled() })
-    def 'should fail AsyncCompletionHandler if name resolution fails'() {
```

Comment:
Why was this test removed? It seems that the behavior tested here is still there, and technically we should still be able to test it. This question is also applicable to the changes in `AsyncSocketChannelStreamSpecification`.

Reply:
There was no good reason.  And not only that, the `@Slow` tests were all broken so after adding these back I also fixed those.  Sorry about that.